packages/core/namespaces/shapes.ts

```diff
@@ -0,0 +1,10 @@
+import namespace from '@rdf-esm/namespace'
+import { NamedNode } from 'rdf-js'
+import { baseUri } from '../env'
+
+type ShapeTerms =
+  'cube-project/create' | 'cube-project/create#CSV' | 'cube-project/create#ExistingCube'
+
+type ShapesNamespace = (term: ShapeTerms) => NamedNode
+
+export const shape: ShapesNamespace = namespace(baseUri + 'shape/') as any
```

Comment:
Like with `cc`, by defining a `shape` namespace builder we prevent typos where it should be used instead of magic string URIs